frontend/src/components/SharedProjectPage/elements.js

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState, useContext } from 'react'
-import { bool, objectOf, string, object, number, shape } from 'prop-types'
+import { bool, objectOf, string, number, shape } from 'prop-types'
 import { useRouter } from 'next/router'
```

Comment:
Can you sort out the merge conflict with these prop-types please? I think `object` is used below so I'm confused.